index.rst

```diff
@@ -59,8 +64,8 @@ The RMS was calculated with a rolling of 10 (50 rolling is approximately 1 s of
 Results
```

Comment:
I also have a suggestion for the paragraph just above. Feel free to disregard it if I understood it wrong:

"The RMS was calculated with a rolling window of 10 (a 50 rolling window is approximately 1 second of movement). To avoid errors in the RMS when introducing data from the slew (the movement before and after the tracking), we performed the analysis starting 0.1 seconds after the beginning and ending 0.1 seconds before the end"

Reply:
I have changed the text for your suggestion